### PR TITLE
Can now manually set cover pictures. #79

### PR DESCRIPTION
**Pull request by NoodleMage**
- Removed FAB library now uses the internal one. 
- Changed getTimestamp to modification date.
- Rewrote IOHandler.  
- Fixed Google Drive bug. 
- Tested working for API 16 and 23
- Fixed merge bugs

### CODE REVIEW

**Comment by NoodleMage** [comment]
ezgif-2841955649

File: app/src/main/java/eu/kanade/tachiyomi/ui/manga/info/MangaInfoPresenter.java [hunk 1]
```diff
@@ -39,22 +68,29 @@ protected void onCreate(Bundle savedState) {
...
                 () -> Observable.just(count),
                 MangaInfoFragment::setChapterCount);
 
+        // Fetch manga info from source
         restartableFirst(FETCH_MANGA_INFO,
                 this::fetchMangaObs,
                 (view, manga) -> view.onFetchMangaDone(),
                 (view, error) -> view.onFetchMangaError());
 
+        // TODO reason?
```
Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/ui/manga/info/MangaInfoPresenter.java]: `onEventMainThread` receives an event thanks to this line. :D

**Comment by inorichi** [comment]
Looks good. :)

I'm not sure if I should move the `IOHandler` to `data/io`.

I'll merge when you tell me.

File: app/src/main/res/layout/fragment_manga_info.xml [hunk 2]
```diff
@@ -238,24 +239,45 @@
...
+        android:gravity="bottom">
+
+        <android.support.design.widget.FloatingActionButton
+            android:id="@+id/fab_edit"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="bottom|right"
+            android:layout_margin="@dimen/fab_margin"
+            app:backgroundTint="@color/colorPrimary"
+            app:layout_anchor="@id/categories_list"
```
Comment (inorichi) [inline app/src/main/res/layout/fragment_manga_info.xml]: Doesn't this break anything?

Comment (NoodleMage) [inline app/src/main/res/layout/fragment_manga_info.xml]: Lol. Bad copy paste xD
Shall fix it. One sec

**Pull request merged**